core/vm/memory.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/holiman/uint256"
 )
 
+const bit32 = 32
```

Comment:
I don't see why a constant `bit32` is easier to read than `32` ?

Reply:
oh, This is my code cleanliness.